### DISCUSSION TITLE
Apk to APKS Maker

### DISCUSSION THREAD

**Discussion by ThanosAtha** [iBotPeaches/Apktool#3294] [Ideas]
Well, I'm dealing with the known Scoped Storage of Android 13, which doesn't let me to copy obb files for games installation... I could install the game as a bundle to avoid this problem, so can I convert apk & obb to .apks installer, and install it with ApkTool M??? It would be really useful & lifesaver!

**Comment by iBotPeaches** [comment]
Apologies - not following what Apktool could do here.
